scripts/coverage/gcov-gen-html

```diff
@@ -82,7 +82,7 @@ then
 		cat << EOF
 		push(@gcov_options, "-a") if ($gcov_caps->{'all-blocks'});
 EOF
-		exit 1
+		#exit 1
```

Comment:
Add a comment that we need this so that we do not have to change anything in gcov tool.

Reply:
done